plugins/alias/src/index.js

```diff
@@ -9,6 +9,7 @@ module.exports = function(context, options) {
         resolve: {
           alias: {
             CSS: path.resolve(__dirname, '../../../src/css'),
+            Components: path.resolve(__dirname, '../../../src/components'),
```

Comment:
Will this allow `import Hero from components/Hero`? If so keep it lowercase.

Reply:
Yes. If hero is the components folder

Reply:
Great. Let's make the alias lowercase. I don't care about CSS being uppercase.